languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -86541,14 +86541,17 @@ The accident victim died from her injuries.
             <rule>
                 <pattern>
                     <marker>
-                        <token>gonna</token>
+                        <token regexp="yes">g[ou]n?na|finna</token>
```

Comment:
"Gunna" is also a rapper:
https://en.wikipedia.org/wiki/Gunna_(rapper)

Reply:
How about an AP like this (to exclude case sensitive "Gunna" except at SENT_START):

```xml
<antipattern>
    <token negate_pos="yes" postag="SENT_START"/>
    <token case_sensitive="yes">Gunna</token>
</antipattern>
```

Reply:
_These allegations concerning Gunna have no basis in reality._